agent/package.ddl

```diff
@@ -248,3 +248,17 @@ action "checkupdates", :description => "Check for updates" do
            :description => "The exitcode from the package manager command",
            :display_as => "Exit Code"
 end
+
+action "refresh", :description => "Update the available packages cache" do
+    output :output,
+           :description => "Output from the package manager",
+           :display_as  => "Output"
+
+    output :outdated_packages,
+           :description => "Outdated packages",
+           :display_as  => "Outdated Packages"
```

Comment:
wonder if refresh really should imply check outdated?

Reply:
It's what `apt_update` did, so I went the same way.  In my use case, I ran `apt_update` once and then `checkupdates` as many times as needed while performing updates.

I was thinking replacing `apt_update` with `refresh` and keeping that behavior, but it's really not needed.  Do you prefer that the rpc only returns an error code and the output from the command and that the application list failed nodes?